Tests/Svg.UnitTests/ExCSSLexerTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using System.Linq;
+
+namespace Svg.UnitTests
+{
+    /// <summary>
+    ///This is a test class for CssQueryTest and is intended
+    ///to contain all CssQueryTest Unit Tests
+    ///</summary>
+    [TestFixture]
+    public class ExCSSLexerTests
+    {
+        [Test]
+        public void LexerStylesheet()
+        {
+            //
+            TestStylesheetReader("#a{fill:#000000;font-family:ZWBIQX+HelveticaUnicodeMS;}", 14);
```

Comment:
I think that check test results.

tokens is thefollowing.
```
font-family
:
ZWBIQX
+
HelveticaUnicodeMS
;
```
Is expected `font-family` value `ZWBIQX+HelveticaUnicodeMS` ?
In that case, I think that it is better to implement it with reference to `-`.